python-rhsm/python-rhsm.spec

```diff
@@ -77,25 +83,32 @@ Python. The name instead reflects its relationship to python-rhsm.
 
 %build
 # create a version.py with the rpm version info
-PYTHON_RHSM_VERSION=%{version} PYTHON_RHSM_RELEASE=%{release} CFLAGS="%{optflags}" LDFLAGS="%{__global_ldflags}" %{__python} setup.py build
+PYTHON_RHSM_VERSION=%{version} PYTHON_RHSM_RELEASE=%{release} LDFLAGS="%{__global_ldflags}" %{py2_build}
+PYTHON_RHSM_VERSION=%{version} PYTHON_RHSM_RELEASE=%{release} LDFLAGS="%{__global_ldflags}" %{py3_build}
 
 %install
 rm -rf %{buildroot}
-%{__python} setup.py install -O1 --skip-build --root %{buildroot} --prefix=%{_prefix}
+%{py2_install}
```

Comment:
Do you have a reference to the macros SUSE has available?  If you wouldn't mind sending me the output of `rpm --showrc` that should have all of them.